sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1468,6 +1489,7 @@ def task_worker(self):
                                self.log_notice("{} Forcing Tx laser OFF".format(lport))
                                # Force DataPath re-init
                                api.tx_disable_channel(media_lanes_mask, True)
+                               self.reset_port_active_apsel_to_db(lport, host_lanes_mask)
```

Comment:
@chiourung Can we reuse `post_port_active_apsel_to_db` function and introduce an additional argument to ignore reading the active application to reduce code replication?